tests/unit/test_speechManager/__init__.py

```diff
@@ -826,10 +826,16 @@ def test_4_profiles(self):
 			"Testing testing ",
 			speech.PitchCommand(offset=100),
 			"1 2 3 4",
-			smi.create_ConfigProfileTriggerCommand(t1, True, expectedToBecomeIndex=1),
-			smi.create_ConfigProfileTriggerCommand(t2, True, expectedToBecomeIndex=2),
+			ExpectedIndex(1),

```

Comment:
Is there a reason not to use `smi.create_expectedIndex` here (and in other places)?

The `smi.create_expectedIndex` function will also check to make sure that you don't accidentally skip or duplicate any indexes. With the goal of avoiding errors in the test. It does mean that you have to be diligent with how the expected indexes are created, which can be annoying too. My logic was that it is better to be annoyed by constraints during development time than to find out some time later that the test didn't do what I thought.